Community-Supported/hyper-jupyter-kernel/hyper_kernel/kernel.py

```diff
@@ -0,0 +1,209 @@
+from ipykernel.kernelbase import Kernel
+from tableauhyperapi import HyperProcess, Connection, Telemetry, HyperException
+from tabulate import tabulate
+import time
+import json
+from datetime import datetime
+import shlex
+
+
+class HyperKernel(Kernel):
+    implementation = 'Hyper'
+    implementation_version = '0.0'
+    language = 'sql'
+    language_version = '0.0'
+    language_info = {
+        'name': 'sql',
+        'mimetype': 'text/sql',
+        'file_extension': '.sql',
+    }
+    banner = "Hyper 🚀 - Your friendly neighborhood SQL database.\n" +\
+             "Type '\\?' for help."
+
+
+    def __init__(self, *args, **kwargs):
+        super(HyperKernel, self).__init__(*args, **kwargs)
+
+        self._hyper_process = HyperProcess(Telemetry.DO_NOT_SEND_USAGE_DATA_TO_TABLEAU, 'jupyter_sql_kernel')
```

Comment:
Why don't we send usage data to Tableau?

Reply:
because we don't have a mechanism for the end-user to opt-in or opt-out of telemetry collection.
Afaik, sending by default without user-agreement isn't legal

Reply:
👍